locales/en-US/snoozetabs.properties

```diff
@@ -1,35 +1,45 @@
-# LOCALIZATION NOTE: This add-on will let you select tabs to close for a while, after which they will re-appear in your browser with a notification.
+# This add-on will let you select tabs to close for a while, after which they
+# will re-appear in your browser with a notification.
 extDesc=An add-on to let you snooze your tabs for a while.
 popupTitle=Snooze tab until…
 contextMenuTitle=Snooze tab until…
 bookmarkFolderTitle=Snoozed Tabs
 notificationTitle=Tab woke up…
 
+# Alternative text for the tab icon
 confirmIconAltText=tab icon
-# LOCALIZATION NOTE: "{time}" is the time the tab will snooze until, as per https://github.com/bwinton/SnoozeTabs/issues/60
-confirmTimeTitle=This tab will snooze until {time}.
+
+# "{time_exp}" is the time the tab will snooze until, see also
```

Comment:
Wrong link, but if you wanted to fix it, I think you could change [this line](https://github.com/bwinton/SnoozeTabs/blob/master/pontoon-to-webext.js#L97) and [this line](https://github.com/bwinton/SnoozeTabs/blob/master/pontoon-to-webext.js#L128), and have it all work.
(Maybe pull that regex out into a constant, and make it `/\{([A-Za-z0-9_@]*)\}/g` as per [the docs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/i18n/Locale-Specific_Message_reference#name)?)